docs/sdks/client-sdks/flutter/initialization.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Initialization
+sidebar_position: 3
+---
+
+The Eppo SDK for Dart and Flutter is easy to initialize while offering robust customization options, 
+making it adaptable to various use cases such as custom caching requirements and ultra-low-latency initialization.
+
+## Initialize the SDK
+
+To complete basic initialization, you only need to provide an SDK key. [Create an SDK key](/sdks/sdk-keys) if you don't already have one.
+
+Add the Eppo Dart SDK to your `pubspec.yaml` file:
+
+```yaml
+dependencies:
+  eppo: ^1.0.0
+```
+
+Then, initialize the SDK:
+
+```dart
+import 'package:eppo/eppo.dart';
+
+final subject = Subject(
+  subjectKey: 'user-identifier',
+  subjectAttributes: ContextAttributes(),
+);
+
+await Eppo.initialize(
+  PrecomputeArguments(subject: subject),
+  SdkOptions(
+    sdkKey: 'your-sdk-key'
+  ),
+);
+```
+
+## Use the SDK instance
+
+After initialization, you can use the SDK singleton to assign a variation to a subject using the assignment methods.
+
+```dart
+final bool variation = Eppo.getBooleanAssignment(
+  '<flag-key>',
+  false,
+);
+```
+
+### Example Configuration
+
+Here's an example of initializing the SDK with custom configuration:
+
+```dart
+import 'package:eppo/eppo.dart';
+
+class YourAssignmentLogger implements AssignmentLogger {
+  @override
+  void logAssignment(AssignmentEvent event) {
+    print('warehouse assignment logged:', event.toJson());
+  }
+}
+
+class YourBanditLogger implements BanditLogger {
+  @override
+  void logBanditEvent(BanditEvent event) {
+    print('warehouse bandit logged:', event.toJson());
+  }
+}
+
+await Eppo.initialize(
+  PrecomputeArguments(subject: subject),
+  SdkOptions(
+    sdkKey: 'your-sdk-key',
+    assignmentLogger: YourAssignmentLogger(),
+    banditLogger: YourBanditLogger(),
```

Comment:
if we have bandits at play I imagine we'd want to pass in actions for at least one bandit as well